### PR TITLE
ci: ignore docs checker in queue

### PR DESCRIPTION
**Pull request by h4ck3rk3y**
## Description:
Ignoring docs checker in the queue as during a release please release it isn't able to find the tag comparisions; and for some reason ignore patterns isn't working

## Is this change user facing?
NO

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -962,7 +962,7 @@ workflows:
               ignore:
                 - main
                 - release-please--branches--main--components--kurtosis
-                - gh-readonly-queue/main/.*
+                - /gh-readonly-queue.*/
```
Comment (gbouv) [inline .circleci/config.yml:956]: Interesting, why does this disable the checker in the queue? Are branches not named `/gh-readonly-queue/main/` something?

**Review by gbouv — APPROVED** [review]

**Comment by h4ck3rk3y** [comment]
Haven’t tried I thought this regular expression would cover that case, the
wrapping // are just regular expression notation

The branches are indeed called

gh-readonly-queue/main/pr-hash

File: .circleci/config.yml (hunk 1 above)
Comment (h4ck3rk3y) [reply .circleci/config.yml:956]: Haven’t tried I thought this regular expression would cover that case, the
wrapping // are just regular expression notation

The branches are indeed called

gh-readonly-queue/main/pr-hash

**Comment by h4ck3rk3y** [comment]
Closing this for now we'll get to this later

**Pull request closed without merging**